extensions/cobalt/package.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "cobalt",
+  "title": "Cobalt",
+  "description": "Download videos and audio from popular social media platforms with ease.",
+  "icon": "command-icon.png",
+  "author": "aelew",
+  "categories": [
+    "Applications",
+    "Data",
+    "Media",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Download Media",
+      "subtitle": "Cobalt",
+      "description": "Download video and/or audio from social media with customizable settings.",
+      "mode": "view"
+    }
+  ],
+  "preferences": [
+    {
+      "name": "downloadDirectory",
+      "title": "Download directory",
+      "description": "Where Cobalt should save downloaded files.",
+      "default": "~/Downloads",
+      "type": "directory",
+      "required": true
+    },
+    {
+      "name": "apiInstanceUrl",
+      "title": "API Instance URL",
+      "description": "Want to use a different Cobalt API instance? Change your instance URL here!",
+      "placeholder": "https://co.wuk.sh",
+      "default": "https://co.wuk.sh",
+      "type": "textfield",
+      "required": true
```

Comment:
Maybe this should be `false` instead, is this setting even necessary?

Reply:
Some people prefer to [self-host](https://github.com/wukko/cobalt/#host-an-instance-yourself) the cobalt API, so this setting will come in handy for them. I'll change it to not be required though since the average user probably won't do this.